Dockerfile-csi-host-definer

```
@@ -0,0 +1,42 @@
+FROM registry.access.redhat.com/ubi8/python-38:1-75.1638364053 as builder
+
+RUN pip3 install --ignore-installed --default-timeout=100 --upgrade pip==21.2.4
+WORKDIR /tmp
+COPY controllers/servers/csi/requirements.txt ./csi_requirements.txt
+COPY controllers/servers/host_definer/requirements.txt ./host_definer_requirements.txt
+# avoid default- boringssl lib, since it does not support z systems
+ENV GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=True
+RUN pip3 install -r ./csi_requirements.txt -r ./host_definer_requirements.txt
+
+USER root
+COPY controllers/scripts/csi_general .
+RUN ./csi_pb2.sh
+RUN pip3 install .
+
+FROM registry.access.redhat.com/ubi8/python-38:1-96
+
+ARG VERSION=1.10.0
+ARG BUILD_NUMBER=0
+
+###Required Labels
+LABEL name="IBM block storage CSI driver host definer" \
+      vendor="IBM" \
+      maintainer="IBM Storage" \
+      version=$VERSION \
+      release=$BUILD_NUMBER \
+      summary="The host definer defines Kubernetes nodes on IBM block storage" \
+      description="The host definer defines container orchestrators, such as Kubernetes and OpenShift, nodes on IBM block storage." \
```

to me it sounds awkward to put a comma between the adjective and its noun. how about:
```suggestion
      description="The host definer defines container orchestrators nodes, such as of Kubernetes and OpenShift, on IBM block storage." \
```
@rivkap what do you think?

---

I think either:

The host definer defines container orchestrators, such as Kubernetes and OpenShift nodes on IBM block storage.

or

The host definer defines container orchestrator nodes on IBM block storage, such as Kubernetes and OpenShift.

---

the 1st is similar to the existing description (minus 1 of the commas), which still sounds awkward to me.
the 2nd sounds to me like Kubernetes and OpenShift are examples of IBM block storage.

@matancarmeli7 could we just drop the `Kubernetes and OpenShift` here?

---

@oriyarde yes

---

You could also update to 
The host definer defines IBM block storage container orchestrator nodes, such as Kubernetes and OpenShift.
or just
The host definer defines container orchestrator nodes on IBM block storage.

---

>IBM block storage container orchestrator nodes

I don't think there is such a thing.

>The host definer defines container orchestrator nodes on IBM block storage.

LGTM